src/main/java/net/greatstart/configs/SecurityConfig.java

```diff
@@ -37,10 +37,10 @@ protected void configure(HttpSecurity http) throws Exception {
                 .loginPage("/user/login")
                 .usernameParameter("email")
                 // TODO: change the routing when home page is ready
-                .defaultSuccessUrl("/project/")
+                //TODO: uncomment this line: .defaultSuccessUrl("/project/")
                 .and()
                 .logout()
-                .logoutSuccessUrl("/project/")
+                //TODO: uncomment this line: .logoutSuccessUrl("/project/")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)